### ISSUE TITLE
[BUG] file.inlinks doesn't work

### ISSUE THREAD

**Issue by SandroGrzicic** [udus122/dataview-publisher#41]
# Title
[BUG] file.inlinks doesn't work in 0.3.7 (dataview 0.5.67)

## Overview
When using file.inlinks in the dataview query, the result is unusable.

## Reproduction procedure
Example query (replace "some-page" with a page that has inlinks, and optionally "some-prefix" (or remove the filter)):

```dataview
TABLE WITHOUT ID
file.link AS "File Link",
sort(filter(file.inlinks, (l) => startswith(meta(l).path, "some-prefix"))) AS "Links" FROM "some-page"
```

## What if I don't fix it?

The output is invalid

## Causes

Presumably something to do with link handling?

## Solutions

Handle the links correctly.

Thank you for the great plugin!

**Comment by udus122** [comment]
@SandroGrzicic Thank you for reporting.

Is the invalid output like below right?

This seems to be caused by the `\` being escaped, like `\\|`.
Changing `\\|` to `\|` may fix the problem.

<img width="643" alt="image" src="https://github.com/udus122/dataview-publisher/assets/22557372/79160623-6a81-4e21-a911-bb260d7c4d3d">

````
%% DATAVIEW_PUBLISHER: start
```dataview
TABLE WITHOUT ID
file.link AS "File Link",
sort(filter(file.inlinks, (l) => startswith(meta(l).path, "Journals/Daily"))) AS "Links"

FROM "Journals"
limit 5
```
%%

| File Link                                    | Links                                                                                                                                                                       |
| -------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| [[Journals/Daily/2024-06-16.md\|2024-06-16]] | <ul><li>[[Journals/Daily/2024-06-17.md\\|2024-06-17]]</li></ul>                                                                                                             |
| [[Journals/Daily/2024-06-17.md\|2024-06-17]] | <ul><li>[[Journals/Daily/2024-06-16.md\\|2024-06-16]]</li><li>[[Journals/Daily/2024-06-17.md\\|2024-06-17]]</li><li>[[Journals/Daily/2024-06-18.md\\|2024-06-18]]</li></ul> |
| [[Journals/Daily/2024-06-18.md\|2024-06-18]] | <ul><li>[[Journals/Daily/2024-06-17.md\\|2024-06-17]]</li><li>[[Journals/Daily/2024-06-19.md\\|2024-06-19]]</li></ul>                                                       |
| [[Journals/Daily/2024-06-19.md\|2024-06-19]] | <ul><li>[[Journals/Daily/2024-06-18.md\\|2024-06-18]]</li><li>[[Journals/Daily/2024-06-20.md\\|2024-06-20]]</li></ul>                                                       |
| [[Journals/Daily/2024-06-20.md\|2024-06-20]] | <ul><li>[[Journals/Daily/2024-06-19.md\\|2024-06-19]]</li><li>[[Journals/Daily/2024-06-21.md\\|2024-06-21]]</li></ul>                                                       |

%% DATAVIEW_PUBLISHER: end %%
````

**Comment by udus122** [comment]
I fixed the above problem and [released it  as version `0.3.8`](https://github.com/udus122/dataview-publisher/pull/43).
Please try it.
If it's a different problem, please let me know.

**Comment by SandroGrzicic** [comment]
Amazing! That fixed it!

The resulting table could have a little more padding on the right side, but I'll just fix that with some CSS. Thank you for your amazing work!
